packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.less

```diff
@@ -0,0 +1,5 @@
+.input-icons-softphone-record-link i {
+  position: absolute;
+  right: 20px;
+  top: 25%;
+}
```

Comment:
No need because it already exist a component ui-kit for to do the same thing
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-clipboard-webcomponent--default

Reply:
thks